riscv-sbi.adoc

```diff
@@ -6,6 +6,9 @@
 include::autogenerated/revision.adoc-snippet[]
 :revremark: This document is stable. Assume anything could still change, but limited change should be expected. Visit http://riscv.org/spec-state for further details.
 :url-riscv: http://riscv.org
+:revdate: 8/2023
+:revnumber: v2.0-rc2-6-g35e723d9cfc
+:revremark: Draft
```

Comment:
This is already updated above (as per Jeff's feedback.)

Reply:
@atishp04 I have updated the PR.

Reply:
This is the oudated code.